Account/models.py

```diff
@@ -1,10 +1,33 @@
+import datetime
+from django.forms import ModelForm
 from django.db import models
 from django.contrib.auth.models import User
+from django.utils import timezone
+
+class Tweet(models.Model):
+    tweet_creator = models.ForeignKey(User, on_delete=models.PROTECT)
+    tweet_text = models.TextField(max_length=280)
+    pub_date = models.DateTimeField('date published')
+    def __str__(self):
+        return "{} - Post - {}".format(self.tweet_creator, self.pk)
+    def was_published_recently(self):
+        return self.pub_date >= timezone.now() - datetime.timedelta(days=1)
+
+    def visible_info(self):
+        if not self.tweet_text:
+            return
+        return self.tweet_text
+    
+class Generate_Tweet(ModelForm):
+    class Meta:
+        model = Tweet
+        fields =['tweet_text']
```

Comment:
Does this Generate_Tweet class do anything? 

I see Generate_Tweet() is getting called in views.py but it looks like it is just calling the Generate_Tweet class in forms.py

Reply:
Yes, this is the form, it is a way of having forms within Models

Reply:
The Generate_Tweet in forms calls this ModelForm in it's own way

Reply:
Looks good.

Reply:
Nevermind just tested without the ModelForm, it turns out we do not need it